### ISSUE TITLE
some frequently repeated code here

### ISSUE THREAD

**Issue by cgrad01** [martiwap/prj-rev-bwfs-dasmoto#3]
could use something like the following:

`body {
  font-family: Helvetica
}`

to decrease the amount of code here. Take a look for other opportunities like this.

https://github.com/martiwap/prj-rev-bwfs-dasmoto/blob/master/dasmoto/css/style.css#L1

**Comment by martiwap** [comment]
yeah you are right. should have done it only once instead of repeating it every element. thanks fro reviewing
